package.json

```diff
@@ -282,7 +278,7 @@
         "react-styleguidist": "^13.1.3",
         "react-tether": "^1.0.5",
         "react-textarea-autosize": "^8.5.3",
-        "regenerator-runtime": "^0.13.3",
```

Comment:
don't know if we technically still need this but not confident enough to remove yet

Reply:
wasn't this one of those things that isnt referenced but is used ?

Reply:
don't know but it's directly imported in the elements: https://github.com/box/box-ui-elements/blob/master/src/elements/content-explorer/ContentExplorer.js#L7

Reply:
im pretty sure i tried to remove this one time but i cant remember what broke when i took it out